tbot/bot.py

```diff
@@ -0,0 +1,72 @@
+import telebot
+import requests
+from PIL import Image
+import io
+import os
+from os.path import join, dirname
+from dotenv import load_dotenv
+
+env_path = join(dirname(__file__),'.env')
+load_dotenv(dotenv_path=env_path)
+TOKEN = os.getenv('TG_TOKEN')
+api_url = os.getenv('API_URL')
+
+bot = telebot.TeleBot(TOKEN)
+telegram_download_link = f'https://api.telegram.org/file/bot{TOKEN}/'
+
+UNSUPPORTED_TYPES = ['audio', 'document', 'sticker', 'video', 'video_note', \
+    'voice', 'location', 'contact', 'new_chat_members', 'left_chat_member', 'new_chat_title', \
+        'new_chat_photo', 'delete_chat_photo', 'group_chat_created', 'supergroup_chat_created', \
+            'channel_chat_created', 'migrate_to_chat_id', 'migrate_from_chat_id', 'pinned_message']
+
+@bot.message_handler(commands=['start'])
+def send_welcome(message):
+    bot.reply_to(message, "Hello! Just send me a picture")
+    
+@bot.message_handler(content_types=UNSUPPORTED_TYPES)
+def default_reply(message):
+    bot.send_message(message.chat.id, "Unfortunately I cannot work with this data type 😢")
+    bot.send_message(message.chat.id, 'Send me a picture 😅')
+ 
+@bot.message_handler(content_types=['text'])
+def reply_text(message):
+    bot.reply_to(message, "I'm not very talkative. Better send me a picture 😅")
```

Comment:
Once we work on the painting recommender system, this will actually be a valid input ;)

Reply:
Yes :)